src/components/metadataFormControls/ModelSingleSelect/ModelSingleSelect.tsx

```diff
@@ -84,6 +87,16 @@ export const ModelSingleSelect = <
         return transform ? transform(allDataMap) : allDataMap
     }, [allDataMap, transform])
 
+    useEffect(() => {
```

Comment:
this is relevant when refreshing the merge page, so that you do not get an infinite loader for target information (e.g. the query could not find anything in the first 10 results that were actually available after sorting out the source items, and then would be stuck in infinite loading state)

Reply:
I think that's OK, but not really needed because all the other components extends the level below it. So it would be quite safe as typescript would've helped you here if you just added it to `SearchableSingleSelect`.